packages/app/src/components/QueryExplanation.tsx

```diff
@@ -0,0 +1,19 @@
+export interface ExplainResult {
+  database: string;
+  table: string;
+  parts: string;
+  rows: string;
+  marks: string;
+}
+
+interface QueryExplanationProps {
+  explainResults?: Array<ExplainResult>;
+}
+
+export function QueryExplanation({ explainResults }: QueryExplanationProps) {
+  if (!explainResults?.length) return null;
+
+  const rows = explainResults[0].rows;
+
+  return rows ? <div className="fs-8">Read Rows: {rows}</div> : null;
```

Comment:
If possible, we're trying to move towards the mantine library instead and away from bootstrap (not an explicit goal to retire BS rn but hopefully we can deprecate its usage over time).

The equivalent is something like `<Text size="xs">`

Reply:
Oh also since this is more of a secondary information component, it'd be nice to slap a `c="gray.4"` on the text component much like https://github.com/hyperdxio/hyperdx/blob/ec937e1522271aadfa9e77fbb80084ef65942cf4/packages/app/src/DBSearchPage.tsx#L142

Reply:
Good to know. Will change. Thanks!